README.md

```diff
@@ -74,7 +74,17 @@ https://user-images.githubusercontent.com/15977946/124654387-0fd3c500-ded1-11eb-
 
 ## What's New
 
+- 2022-10-14: MMPose \[v1.0.0rc0\] is released. Major updates include:
```

Comment:
- 2022-10-14: MMPose [v1.0.0rc0](https://github.com/open-mmlab/mmpose/releases/tag/v1.0.0rc0) is released. Major updates include: